templates/bot/js/command-and-response/README.md

```diff
@@ -53,34 +57,70 @@ A helloworld command handler is generated in `bot/src/helloworldCommandHandler.j
 
-After finish development and to distribute your app to others, you can [Publish Teams apps using Teams Toolkit](https://docs.microsoft.com/microsoftteams/platform/toolkit/publish).
+### Build adaptive card with dynamic content
 
-## Reference
+Adaptive card provides [Template Language](https://docs.microsoft.com/en-us/adaptive-cards/templating/) to allow users to render dynamic content with the same layout (the template). For example, use the adaptive card to render a list of items (todo items, assigned bugs, etc) that could varies according to different user.
 
-[Teams Toolkit Command Response Bot](https://aka.ms/teamsfx-command-response)
+1. Add your adaptive card template JSON file under `bot/adativeCards` folder
+1. Import the card template to you code file where your command handler exists (e.g. `myCommandHandler.ts`)
+1. Model your card data
+1. Use `MessageBuilder.attachAdaptiveCard` to render the template with dynamic card data
 
-[Teams Bot Command Menus](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/create-a-bot-commands-menu?tabs=desktop%2Cjavascript)
+### Connect to existing API
 
-[Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
+You usually want to access data or information when building Teams application. If you do not have an appropriate SDK that helps you make an API request, Teams Toolkit is here to help you bootstrap sample code which handles authentication for your API requests. For more information, you can visit [Connect existing API document](https://aka.ms/teamsfx-connect-api).
 
-[Teams Toolkit and Step-by-step Documentations](https://docs.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-fundamentals)
+### Add single sign on
 
-[Teams Toolkit CLI](https://docs.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli)
+If you are responding to a command that need to access user data or information, you can leverage single sign on to provide a smooth experience. Read more about how Teams Toolkit can help you [add sso](https://aka.ms/teamsfx-add-sso) incrementally to your command and response bot.
 
-[TeamsFx SDK](https://docs.microsoft.com/microsoftteams/platform/toolkit/teamsfx-sdk)
+### Edit Teams App manifest
+
+You can find the Teams app manifest in `templates/appPackage/manifest.template.json`.
+
+The file contains template arguments with `{...}` statements which will be replaced at build time. You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/microsoftteams/platform/resources/schema/manifest-schema) for more information.
+
+### More development documentations
+
+* Manage [multiple environments](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/teamsfx-multi-env)
```

Comment:
We can remove the locale info (/en-us) in the docs URL for globalization (e.g.  use https://docs.microsoft.com/microsoftteams/platform/toolkit/teamsfx-multi-env instead).

Reply:
Updated, thanks!